content/v2.1/en/quick-start/jenkinsfile-out-of-scm.md

```diff
@@ -1,104 +1,101 @@
+2.1. Log in KubeSphere with `project-regular` account please refer to [Create Credentials](../../devops/credential) for DockerHub and Kubernetes (KubeConfig), input name as `dockerhub-id` and `demo-kubeconfig` respectively.
 
-2. Create a Java token and copy it, please refer to [Access SonarQube](../../devops/sonarqube).
+2.2. Create a Java token and copy it, please refer to [Access SonarQube](../../devops/sonarqube).
 
-3. Enter devops-demo's project in KubeSphere. Similarly, click **Create** in the Credential, name the credential ID as `sonar-token`, choose `secret_text` as its type, paste the SonarQube token that we copied from the previous step, then click Confirm.
+2.3. Enter devops-demo's project in KubeSphere. Similarly, click **Create** in the Credential, name the credential ID as `sonar-token`, choose `secret_text` as its type, paste the SonarQube token that we copied from the previous step, then click Confirm.
 
 At this point, the three credentials have been created, which will be used in the pipeline.
 
 ![Create Credentials](https://pek3b.qingstor.com/kubesphere-docs/png/20200221223754.png)
 
-### Create Pipeline
+### Step 3: Create Pipeline
 
 Follow with the steps below to create a pipeline using graphical editing panel.
 
-#### Step 1: Fill in the basic information
+#### Fill in the basic information
 
-1. In the DevOps project, select the **Pipeline** on the left and click **Create**.
+3.1. In the DevOps project, select the **Pipeline** on the left and click **Create**.
 
 ![Create Pipeline](https://pek3b.qingstor.com/kubesphere-docs/png/20200221225029.png)
 
-2. In the pop-up window, name it `graphical-pipeline`, click **Next**.
+3.2. In the pop-up window, name it `graphical-pipeline`, click **Next**.
 
+#### Advanced Settings
 
-#### Step 2: Advanced Settings
-
-1. Click **Add Parameter** to add **3** string parameters as follows. These parameters will be used in the docker command of the pipeline. Click **OK** when you've done.
+3.3. Click **Add Parameter** to add **3** string parameters as follows. These parameters will be used in the docker command of the pipeline. Click **OK** when you've done.
 
 | Parameter Type | Name | Default Value | Description |
 | --- | --- | --- | --- |
 | String  | REGISTRY | The sample repository address is  docker.io. | Image Registry |
 | String  | DOCKERHUB_NAMESPACE | Fill in your  DockerHub account (which can also be the Organization name under the account). | DockerHub Namespace |
 | String parameter | APP_NAME | Fill the application name with `devops-sample`. | Application Name |
 
-
 ![Advanced Settings](https://pek3b.qingstor.com/kubesphere-docs/png/20200222155944.png)
 
-### Editing pipeline
+### Step 4: Editing pipeline
 
 This pipeline consists of six stages, we will demonstrate steps and tasks in each stage.
 
-#### Stage 1: Pull Source Code (Checkout SCM)
```

Comment:
The Stage 1 ~ 6 are related to the pipeline stage, it might be better to put them in order.

Reply:
It might cause user because of step and stage together here.